packages/gatsby-cli/src/create-cli.js

```diff
@@ -326,6 +326,11 @@ module.exports = argv => {
       describe: `Turn off the color in output`,
       global: true,
     })
+    .option(`logger`, {
+      describe: `Define a logger for output`,
+      choices: [`json`, `ink`, `yurnalist`],
```

Comment:
Thought about this some more and after speaking to @pieh, can we instead just expose `--json` instead of `logger` since we likely don't want to expose both `ink` and `yurnalist`? (as they are internal implementation details that we want to be able to change without a breaking version)

Reply:
Yeah, absolutely. I'll make that change in the next couple days. Thanks for taking a look!

Reply:
Should be updated to only expose JSON logger now. LMK if anything else is needed! 🍻

Reply:
You're a total champ! Thank you so much 🤗